blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DatastaxIO.java

```diff
@@ -57,13 +57,19 @@ private static void connect() {
         CodecRegistry codecRegistry = new CodecRegistry();
 
         cluster = Cluster.builder()
-                .withLoadBalancingPolicy(new TokenAwarePolicy(DCAwareRoundRobinPolicy.builder().withLocalDc("datacenter1").build()))
+                .withLoadBalancingPolicy(new TokenAwarePolicy(DCAwareRoundRobinPolicy.builder().withLocalDc("datacenter1").build(), false))
```

Comment:
This is to turn off shuffleReplica flag, per documentation here:
http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/core/policies/TokenAwarePolicy.html

This is recommended by the guys from Last Pickle (see slide 25)
http://www.slideshare.net/planetcassandra/successful-software-development-with-apache-cassandra-54922569